src/Mod/TechDraw/App/ArrowPropEnum.cpp

```diff
@@ -30,14 +30,14 @@
 namespace TechDraw {
 
 const int   ArrowPropEnum::ArrowCount = 8;
-const char* ArrowPropEnum::ArrowTypeEnums[]= { "Filled Arrow",
-                               "Open Arrow",
-                               "Tick",
-                               "Dot",
-                               "Open Circle",
-                               "Fork",
-                               "Filled Triangle",
-                               "None",
+const char* ArrowPropEnum::ArrowTypeEnums[]= { QT_TRANSLATE_NOOP("TechDraw", "Filled Arrow"),
```

Comment:
Not sure if it is a misuse of the term "enum". Would like to know more if it isn't.